wrapper_testing.py

```diff
@@ -0,0 +1,54 @@
+"""Testing file for the wrappers."""
+import numpy as np
+
+import momaland.utils.aec_wrappers as AECWrappers
+import momaland.utils.parallel_wrappers as ParallelWrappers
+from momaland.envs.multiwalker import momultiwalker_v0 as _env
+
+
+def parallel_test():
+    """Full ParallelEnv lifecycle for testing wrappers."""
+    env = _env.parallel_env(shared_reward=False)
+    env = ParallelWrappers.LinearizeReward(env, np.array([0.8, 0.15, 0.15]))
+    env = ParallelWrappers.NormalizeReward(env, env.possible_agents[0], [0])
+    env = ParallelWrappers.NormalizeReward(env, env.possible_agents[1], [0])
+    env = ParallelWrappers.NormalizeReward(env, env.possible_agents[2], [0])
+
+    _, _ = env.reset(seed=42)
+
+    while env.agents:
+        # this is where you would insert your policy
+        actions = {agent: env.action_space(agent).sample() for agent in env.agents}
+
+        _, rewards, _, _, _ = env.step(actions)
+        for key, value in rewards.items():
+            print(key, value)
```

Comment:
I guess you can test: 

- That you have only one reward component in case it was linearized
- That rewards are within [-1, 1] is normalized ?

Reply:
linearized test is good
normalized test is also good, although for parallel it can be out of bounds, still much lesser than the raw reward though

Reply:
Testing I meant assert, not printing 🗡️